stable/Dockerfile

```diff
@@ -59,6 +59,9 @@ RUN apt-get update -y && \
   socat \
   xvfb \
   x11vnc \
+  openssh-server \
+  telnet \
+  libgtk-3-0 -y \
```

Comment:
to run TWS in the container libgtk 3.0 is the only necessary library to add

Reply:
libgtk 3.0 was not the only thing i needed in my test. that's strange.

in any case, I do like the idea of having TWS, but to be honest, I don't think this is a good 'user experience'. there is no desktop, windows manager, no browser, it rather limited.

Please take these lines our of the PR. I think this needs more analysis. I'm thinking on implementing the change using a different base image, something like [docker-rdesktop](https://github.com/linuxserver/docker-rdesktop). we can run IBC on that image with a fully functional desktop environment.

One important point, please take out all changes on folders stable and latest. Changes should be on the dockerfile.template or on image-script/* forlder.

Reply:
@nalon99 , could you please check #45 ? I have explored the path of [docker-rdesktop](https://github.com/linuxserver/docker-rdesktop) and i think it looks promising. I would appreciate your feedback.

Reply:
rdp protocol is faster and more secure than standard vnc protocol and rdp clients are available also for Linux and Mac os. It would be a good chance to use it.

Reply:
> rdp protocol is faster and more secure than standard vnc protocol and rdp clients are available also for Linux and Mac os. It would be a good chance to use it.

I've been pushing commits into #45  branch and it's almost ready. if you can please test it, i would appreciate your tests and feedback. the code is ready for building a local image for "latest", basically just use `tws-docker-compose.yml` file

by the way, I'm able to run tws with IBC and "activeX and socker client" are enabled by default, I don't need to do anything to have the API working

the way I see this is as a different image, one for ibgateway and one for tws. with a common logic for IBC, socat and ssh tunnels.